src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -766,6 +785,9 @@ private static ClassLoader contextClassLoader(Thread parent) {
             this.contextClassLoader = ClassLoader.getSystemClassLoader();
         }
 
+        // special value to mean a new thread
+        this.scopedValueBindings = NEW_THREAD_BINDINGS;
```

Comment:
Can we change the comment on this one to be the same as the other constructor?

Reply:
Done.